components/src/core/empty-state/empty-state.tsx

```diff
@@ -131,7 +131,7 @@ export const EmptyState: React.FC<EmptyStateProps> = (props) => {
 
     const getColor = useCallback(
         (color: string | undefined): string => {
-            if (!color) return theme.colors.textSecondary || theme.colors.text;
+            if (!color) return theme.dark ? PXBColors.black[200] : PXBColors.gray[500];
```

Comment:
Will setting both light and dark themes to `theme.colors.textSecondary` work?

Reply:
theme.colors.textSecondary is the wrong color for light theme. I could still use it for theme definition for dark theme as that one is correct, but figured it best to just hardcode it so we can see the diff going into 6.0.0. We will be using theme definitions for v6 (this work is already done).

Reply:
Asking only because I saw the color is now defined in 6.0.0: https://github.com/pxblue/react-native-themes/blob/release/6.0.0/src/blueTheme.ts#L54